scripts/module/money_drop.nut

```diff
@@ -0,0 +1,30 @@
+alive <- array(100, null);
+MoneyPickups <- {};
+
+function onPlayerSpawn( player ) {
+  alive[player.ID] = true;
+  player.Cash = 500;
+}
+
+function onPlayerPart( player, reason ) {
+  alive[player.ID] = null;
+}
+
+function onPlayerHealthChange( player, lastHP, newHP ) {
+  if ( newHP < 0.1 && alive[player.ID]) {
+    alive[player.ID] = false;
+    local pickup = CreatePickup( 337, player.World, player.Cash * 0.8, player.Position, 255, true);
```

Comment:
The original game drops a lot of pickups around dead people.

I'd like to mimic this behaviour, with a lot of cash icons around the player (10 or something).